pom.xml

```diff
@@ -188,6 +189,27 @@
         <!-- mvn com.mycila:license-maven-plugin:format (add headers if missing) -->
         <!-- mvn com.mycila:license-maven-plugin:remove (remove existing header) -->
         <plugins>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>1.17.0</version>
+                <configuration>
+                    <java>
+                        <trimTrailingWhitespace/>
+                        <removeUnusedImports/>
+                        <includes>
+                            <include>sdk-generate-entity-models-maven-plugin/src/**/java/**/*.java</include>
+                        </includes>
+                        <eclipse>
```

Comment:
I like the idea of using a code formatter, however will this work (integrate) with IntelliJ, we use that internally?

Reply:
https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter

Reply:
we also use intellij at emundo 😉 and all our projects has a build step in our ci to enforce correct formatting. 
https://github.com/diffplug/spotless/tree/master/plugin-gradle
https://github.com/diffplug/spotless/tree/master/plugin-maven